packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -22,6 +22,11 @@
   "force_lightning_event_create_text": "SFDX: Create Lightning Event",
   "force_lightning_interface_create_text": "SFDX: Create Lightning Interface",
   "force_source_status_local_text": "SFDX: View Local Changes",
-  "force_source_status_remote_text": "SFDX: View Changes in Default Scratch Org",
-  "force_debugger_stop_text": "SFDX: Stop Apex Debugger Session"
+  "force_source_status_remote_text":
+    "SFDX: View Changes in Default Scratch Org",
+  "force_debugger_stop_text": "SFDX: Stop Apex Debugger Session",
```

Comment:
@ruthemmanuelle could you verify that these messages are ok? The last one has ellipses to indicate that there will be input required to indicate which org. This seems to be a common trend in other VSCode commands/extensions (i.e. `Files: Open...`, `Git: Push to...`, etc)